manager/dispatcher/dispatcher.go

```diff
@@ -121,7 +121,6 @@ func New(cluster Cluster, c *Config) *Dispatcher {
 		cluster:                   cluster,
 		mgrQueue:                  watch.NewQueue(16),
 		keyMgrQueue:               watch.NewQueue(16),
-		lastSeenManagers:          getWeightedPeers(cluster),
```

Comment:
@LK4D4 I had to remove that line as we now initialize the raft server after starting the dispatcher. Do you know if this has any implication on the current way the dispatcher works?